src/triton_gazebo/models/import_mesh/convert_mesh.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+
+
+import bpy
```

Comment:
Is bpy added as a dependency of the package? If not you should add it  in the package.xml as a system dependency

Reply:
bpy is the Blender API import_mesh.py uses to import .STL files and export them as .dae without opening the GUI, it's in a separate script because the latest Blender uses Python 3.7 and bpy is only compatible up to that release and Ubunutu 20.04 uses 3.8.